addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -314,7 +314,7 @@ def _get_invoice_line_vals(self, line, taxes_vals):
         total_fixed_tax_amount = sum([
             vals['amount']
             for vals in allowance_charge_vals_list
-            if vals['allowance_charge_reason_code'] == 'AEO'
+            if vals.get('charge_indicator')
```

Comment:
Yeah, I definitely was not thinking to do so after moving the test to the proper module, will keep in mind for future PRs :+1:, I ran them to confirm before pushing here and they all passed- hopefully this will be the final commit amendment!

Reply:
no worries, it often happens with this module since tests are only run during staging :)